caravel/views.py

```diff
@@ -1219,14 +1256,12 @@ def favstar(self, class_name, obj_id, action):
     @has_access
     @expose("/slice/<slice_id>/")
     def slice(self, slice_id):
```

Comment:
Okay, our main objective it to auto-generate the url from data stored in the db (ideally without touching slice params because they are hairy). 

so we could remove this method and generate the `/explore/<datasource_type>/<datasource_id>/<slice_id>/` and everything should still work. are you okay with that?